lib/minitest.rb

```diff
@@ -46,6 +46,10 @@ module Minitest
   self.info_signal = "INFO"
 
   ##
+  # Access the testopts for example to use the seed value in your tests.
+  mc.send :attr_accessor, :testopts
```

Comment:
"testopts" isn't a word, nor are they specific to testing. Please switch this to "option" (EDIT: singular).

Reply:
You only access them one at a time.